packages/compass-aggregations/src/components/stage-toolbar/stage-operator-select.tsx

```diff
@@ -61,32 +74,35 @@ export const StageOperatorSelect = ({
     },
     [onChange, index]
   );
+  const portalRef = React.useRef<HTMLDivElement | null>(null);
 
   return (
-    <Combobox
-      value={selectedStage}
-      disabled={isDisabled}
-      aria-label="Select a stage operator"
-      onChange={onStageOperatorSelected}
-      size="default"
-      clearable={false}
-      data-testid="stage-operator-combobox"
-      className={comboboxStyles}
-      // Used for testing to access the popover for a stage
-      popoverClassName={`mongodb-compass-stage-operator-combobox-${index}`}
-    >
-      {stages.map((stage, index) => (
-        <ComboboxOption
-          data-testid={`combobox-option-stage-${stage.name}`}
-          key={`combobox-option-stage-${index}`}
-          value={stage.name}
-          displayName={stage.name}
-          description={
-            (isAtlasOnly(stage.env) ? 'Atlas only. ' : '') + stage.description
-          }
-        />
-      ))}
-    </Combobox>
+    <React.Fragment>
```

Comment:
As far as I can tell, this would be the first of its kind in the codebase.
I suggest using the shorthand:
```suggestion
    <>
```

Reply:
We are already using it at other places, but probably as `Fragment` and not `React.Fragment`.

Reply:
actually there are instances of both in the codebase, not many but they do exist.

Reply:
I searched the wrong codebase when making that comment 🤦

Reply:
For what it's worth: I see no occurrences of `<React.Fragment>`, two of `<Fragment>` and 183 `<>` 🤷